### ISSUE TITLE
Keyboard focus indicator is not visible on the table header and row controls on File management tab.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#1365]
**Describe the bug**
When we navigate with tab key on the table headers of the table, keyboard focus indicator is not visible around it.
If keyboard focus indicator is not visible on the table controls, then keyboard dependent users will not get the information of the current focused element and will be unable to navigate and access the controls within the table properly.

**To Reproduce**
1.Open [URL](https://blazor.radzen.com/datagrid-single-selection) in edge dev browser.
2.Navigate to the table present on the page using tab key.
3.Navigate the table header using tab key and observe that the keyboard focus indicator is visible on table header or not.

Alternatively link your repo with a sample project that can be run.
[URL](https://blazor.radzen.com/datagrid-single-selection)

**Expected behavior**
Proper Keyboard focus indicator should be visible on the table header controls.

**Desktop (please complete the following information):**
OS Build: Windows 11 Enterprise 22H2 (23620.1000)
Edge Dev Browser: Version 122.0.2365.16 (Official build) dev (64-bit)

**Additional context**
Issue is also repro for the table headers on Result table in the search tab.

**Attachments**

https://github.com/radzenhq/radzen-blazor/assets/93735775/9fba7192-c48c-4dee-b922-9e068d962907

**Comment by enchev** [comment]
Fixed in https://github.com/radzenhq/radzen-blazor/commit/347942668d9003788403ec4e315ae2bfcec68263
